server/api.js

```diff
@@ -1,9 +1,32 @@
 import { Router } from "express";
+import pool from "./db";
 
 const router = Router();
 
 router.get("/", (_, res) => {
 	res.json({ message: "Hello, world!" });
 });
 
+router.get("/users", async (req, res) => {
+  try {
+    const users = await pool.query("SELECT * FROM users");
+    return res.json(users.rows);
+  } catch (err) {
+    console.error(err.message);
+  }
+});
+
+
+router.get("/cohorts", async (req, res) => {
+  try {
+    const cohorts = await pool.query("SELECT * FROM cohorts");
```

Comment:
Instead of selecting ``*`` would it be possible to only select certain columns from the database?

Reply:
I think we would need something like "either id or number" from cohorts INNER JOIN name FROM regions  WHERE cohorts.regions_id = regions.id; so that you get cohort number or id with the corresponding regions.